appveyor.yml

```diff
@@ -61,7 +61,7 @@ install:
     - cmd: conda update --yes --quiet conda
 
     # Conda build tools.
-    - cmd: conda install --yes --quiet obvious-ci --channel conda-forge
+    - cmd: conda install --yes --quiet obvious-ci --channel conda-forge msinttypes
```

Comment:
@jswhit you will notice that Python 3.5 builds are fine because the Visual Studio used for Python 3.5 does have `stdint.h`

Here I am adding the `msinttypes` packages that should provide those. Not sure if I need to set some env var to ensure that `netcdf4-python` finds it.